sql-cli/include/base/config/default/configuration.yml

```diff
@@ -7,19 +7,15 @@ connections:
     schema:
     login:
     password:
-
 ## Example of Bigquery connection
-#  - conn_id: bigquery_conn
-#    conn_type: bigquery
-#    description: null
-#    extra:
-#      project: "my-gcp-project"
-#    host: null
-#    login: null
-#    password: null
-#    port: null
-#    schema: null
-
+  - conn_id: bigquery
+    conn_type: bigquery
+    description:
+    host:
+    login:
+    password:
+    port:
+    schema:
```

Comment:
Do we want to have this as a pre-defined connection for the end-users - or should we comment it out - as before? Would this work if they didn't have BQ/GCP providers?

Reply:
> Would this work if they didn't have BQ/GCP providers?

We currently install all providers i.e. the user does not have a choice.

Reply:
Yes, since we install astro-sdk-python[all], the providers would be available.